### ISSUE TITLE
IDFA usage causing App Store rejection

### ISSUE THREAD

**Issue by vmccurley** [alebianco/ANE-Google-Analytics#59]
Apple is getting strict about apps using the IDFA (iOS Advertising Identifier) without actually displaying advertising. It looks like Google Analytics is using the IDFA.

Apparently Google has created a fix where it's possible to remove AdSupport.framework dependancy.
https://code.google.com/p/analytics-issues/issues/detail?id=387

Would it be possible to add a boolean parameter to enable/disable the AdSupport.framework dependancy in this ANE?

**Comment by alebianco** [comment]
No, it won't be possible to include it or exclude it at runtime but I can look at if it's possible to build the extension without that framework. 
Eventually it will go into the next release so it won't be quick I fear... 

edit: Added the task on the trello board https://trello.com/c/tIJluehh/48-ios-remove-dependency-from-adsupport-framework
